backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/functions/Docs_Sales_Invoice_Details_local_currency_conversion.sql

```diff
@@ -34,80 +34,100 @@ CREATE OR REPLACE FUNCTION de_metas_endcustomer_fresh_reports.Docs_Sales_Invoice
 AS
 $$
 
+WITH records as (
+    SELECT it.C_Invoice_ID,
+           CASE
+               WHEN i.m_inout_id IS NULL OR i.c_order_id IS NULL   THEN FALSE --aggregated ICs aren't printed
+               WHEN i.isprintlocalcurrencyinfo = 'N'               THEN FALSE
+               WHEN bp.isprintlocalcurrencyinfo = 'N'
+                   AND   i.isprintlocalcurrencyinfo <> 'Y'        THEN FALSE
+                                                                   ELSE TRUE
+           END             AS IsPrintLocalTaxReporting,
+           cu.iso_code AS local_iso_code,
+           c.c_currency_id AS currency_from,
+           cu.c_currency_id AS currency_to,
+           CASE
+               WHEN i.m_inout_id IS NULL OR i.c_order_id IS NULL THEN NULL --aggregated ICs aren't printed
```

Comment:
why `OR` and not `AND`?

anyways, IMHO, just checking c_order_id is null would be enough.